ocs_ci/ocs/ui/views.py

```diff
@@ -360,6 +360,12 @@
         'a[data-test-id="horizontal-link-Bucket Class"]',
         By.CSS_SELECTOR,
     ),
+    "capacity_breakdown_options": (
+        'button[class="pf-c-select__toggle"]',
+        By.CSS_SELECTOR,
+    ),
+    "capacity_breakdown_projects": ("//button[text()='Projects']", By.XPATH),
```

Comment:
Please try to use CSS Selector wherever possible.

Reply:
I have an issue to find the element with CSS